### DISCUSSION TITLE
10 pacotes ESSENCIAS para JavaScript

### DISCUSSION THREAD

**Discussion by belmirofss** [frontendbr/forum#2362] [Dicas]
Muitas vezes, principalmente desenvolvedores no início de carreira, tem muitas dificuldades em encontrar bons pacotes que podem utilizar. 

[Escrevi um artigo com 10 pacotes MUST HAVE para a maioria dos projetos JavaScript](https://belmirofss.medium.com/10-pacotes-essencias-para-javascript-9a37297f4bc6)

**Comment by gustvoassuncao** [comment]
Muito massa o artigo! Li agora, ficou show! Uso muito o Axios e o Jest, os outros vou conhecer melhor, agora! Valeu!!
